projects/core/styles/theme/wrapper/floating.less

```diff
@@ -0,0 +1,18 @@
+@import '../../taiga-ui-local.less';
+
+/* stylelint-disable order/order */
```

Comment:
maybe disable this rule ?

Reply:
https://github.com/taiga-family/linters/pull/58